### PR TITLE
correct various typos

### PR DESCRIPTION
**Pull request by ReblochonMasque**
Several minor typos were corrected

### CODE REVIEW

File: docs/usage/usage_advanced_debug.rst [hunk 1]
```diff
@@ -64,11 +64,11 @@ not be sent the message. Putting a try/except clause around every sendMessage is
 typically not practical. 
 
 Since exceptions are common during application development (bugs due to 
-invalid arguments, failed assertions, etc), Pypubsub provdes a hook to register
+invalid arguments, failed assertions, etc...), Pypubsub provides a hook to register
```
Comment (swt2c) [inline docs/usage/usage_advanced_debug.rst]: I would argue there should just be a single period here after the etc.  An ellipsis (...) is somewhat redundant with etc.

File: docs/usage/usage_advanced_debug.rst [hunk 2]
```diff
@@ -64,11 +64,11 @@ not be sent the message. Putting a try/except clause around every sendMessage is
...
 a 'listener exception' handler: whenever a listener raises an exception, 
 Pypubsub then sends it to the handler, and continues with the send operation
 until all listeners have received the message. The handler might print it to
-a log file, output a message in a status bar, show an error box, etc. The 
+a log file, output a message in a status bar, show an error box, etc... The
```
Comment (swt2c) [inline docs/usage/usage_advanced_debug.rst]: Same here, I would not add an ellipsis.

File: docs/usage/usage_advanced_debug.rst [hunk 3]
```diff
@@ -78,14 +78,14 @@ of the handler can be given to ``pub.setListenerExcHandler()``.
...
-Pypubsub defines a specia topic named pub.ALL_TOPICS. A listener that subscribes to
+Pypubsub defines a special topic named pub.ALL_TOPICS. A listener that subscribes to
 this topic will receives all messages of every topic. By default, the listener
 will not receive any data since pub.ALL_TOPICS is the parent of all root topics: 
 its MDS must be empty. 
 
 However, any listener that is a callable
 with a "catch-all" \**kwargs parameter will be given all message data. Moreover, 
-Pypubsub sends the topic object automatically with the message data if it
+Pypubsub sends the topic object automatically with the message data, if it
```
Comment (swt2c) [inline docs/usage/usage_advanced_debug.rst]: I don't think a comma is appropriate here.

**Comment by ReblochonMasque** [comment]
Noted: I noticed the missing chars, corrected them, and thought that I'd scan the entire page for other things to correct in order to do a thorough job, and I think I got carried away trying to find stuff, sorry.
I reverted the three changes you pointed at.

**Comment by schollii** [comment]
Thanks I will check this once I get back from vacation next week

**Comment by schollii** [comment]
Thanks @ReblochonMasque for fixes and @swt2c for reviewing, much appreciated!

**Pull request merged**